test_data/rules/package_prefixed_library_names.dart

```diff
@@ -4,4 +4,5 @@
 
 // test w/ `dart test -N package_prefixed_library_names`
 
-library linter.not_where_it_should_be; //LINT
+// See: https://github.com/dart-lang/linter/issues/3395
+library linter.not_where_it_should_be; //FAILING
```

Comment:
Why would commenting out code that isn't being run cause the test to start failing? How confident are we that the commented out code really isn't being run?

Reply:
It *is* being run in the linter tests since they do take the code path that initializes project instances.  That's however the only path that does and so the tests here are out of sync w/ how server and the analyzer CLIs work.  Obviously, not good!

Reply:
Related: https://github.com/dart-lang/linter/issues/2969